packages/frontend-shared/src/components/Select.cy.tsx

```diff
@@ -101,13 +101,13 @@ describe('<Select />', () => {
   })
 
   describe('#icons', () => {
-    // TODO: Enable with completion of UNIFY-1375
```

Comment:
@marktnoonan @warrensplayer can we close this issue now or is it already closed?